docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,29 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a Lodestar beacon with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) for the beacon node to propose. 
+
+With produceBlockV3 (enabled automatically after the Deneb hard fork), the `builder.boostFactor` is a percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `maxprofit`. Even though this is set on the validator client, the calculation is applied on the beacon node itself. For more information, see the [produceBlockV3 Beacon API](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlockV3).
```

Comment:
How did you decide when to prefix with `--` and when not? I think here we might wanna prefix `builder.boostFactor` as we also did it for `--builder.selection`. Not sure further below, might be fine without  as it is a bit redundant, people know at this point that we talk about CLI flag

Reply:
Good point, consistency is key here. Looking at other parts of our documentation, when explicitly talking about flags, there are `--flag` in the docs. Options are often written as `option` Will address.